kubeflow/common/cluster/upstream/cluster.yaml

```diff
@@ -45,14 +45,22 @@ spec:
   releaseChannel:
     # Per https://github.com/GoogleCloudPlatform/k8s-config-connector/issues/194
     # use upper case for the channels
-    channel: REGULAR
+    channel: STABLE
+  # Master version controls the kubernetes API version.
+  # Future upgrade of master version might break Kubeflow deployment.
+  # At the time of writing, STABLE release channel is using 0.17.
```

Comment:
Nit: 1.17

Reply:
Thank you Yuan for the good catch! Fixed.